src/styles/index.module.css

```diff
@@ -0,0 +1,63 @@
+.container {
+    padding-top: 100px;
+    height: 90vh;
+    margin: 2rem;
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr;
+    grid-gap: 50px;
+  }
+  .cell {
+    /* padding-top: 100px; */
+    width: 200px;
+    height: 250px;
+    left: 160px;
+    top: 198px;
+
+    background: #000000;
+    border: 3px solid #000000;
+    box-sizing: border-box;
+  }
+
+  .cellTitle {
+    /* position: absolute;
+    width: 176px;
+    height: 59px;
+    left: 322px;
+    top: 664px; */
+
+    font-family: Roboto;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 20px;
+    line-height: 59px;
+    /* identical to box height */
+    text-align: center;
+    color: #FFFFFF;
+
+  }
+
+  .circlePic {
+    padding-left: 37px;
```

Comment:
the circle picture is centered statically atm, which is non-ideal (any changes to the container will require this code to be updated). Please find a different way to position the image